packages/sdk/src/controllers/LayoutController.ts

```diff
@@ -215,4 +215,52 @@ export class LayoutController {
         const res = await this.#editorAPI;
         return res.resetLayoutIntent(id).then((result) => getEditorResponseData<null>(result));
     };
+
+    /**
+     * This method sets the bleed value of a specific layout.
+     * Note: this is only valid on a print layout
+     *
+     * @param id The id of the specific layout
+     * @param value The bleed value
+     * @param position When defined will update the bleed value of a single position,
+     * otherwise will set all positions to the same value.
```

Comment:
Should we add a comment that this implies (or does not) areBleedValuesCombined change